gradle/libs.versions.toml

```diff
@@ -76,7 +76,7 @@ plugin-kotlin = { module = "org.jetbrains.kotlin:kotlin-gradle-plugin", version.
 plugin-detekt = { module = "io.gitlab.arturbosch.detekt:detekt-gradle-plugin", version.ref = "detekt" }
 plugin-android = "com.android.tools.build:gradle:7.4.0"
 
-detekt-compose = "com.twitter.compose.rules:detekt:0.0.26"
+detekt-compose = "io.nlopez.compose.rules:detekt:0.1.5"
```

Comment:
Twitter no longer updates this (due to the maintainers no longer working there).
This is the most up to date fork from the most active maintainer of the project.